packages/amazonq/.changes/next-release/Removal-319fc7c8-22e0-44f1-a636-2f4040689b32.json

```diff
@@ -0,0 +1,4 @@
+{
+	"type": "Removal",
+	"description": "Reverted prefetch logic to enable more stable inline completion"
```

Comment:
These changes haven't been released right? If so, just remove the old changelog + this one